code/__DEFINES/dcs/signals/signals_atom/signals_atom_main.dm

```diff
@@ -55,6 +55,8 @@
 #define COMSIG_ATOM_SET_SMOOTHED_ICON_STATE "set_smoothed_icon_state"
 ///from base of [/atom/proc/smooth_icon]: ()
 #define COMSIG_ATOM_SMOOTHED_ICON "atom_smoothed_icon"
+/// from base of whenever the smoothing needs to go: ()
+#define COMSIG_ATOM_REMOVE_SMOOTHING "atom_remove_smoothing"
```

Comment:
this isn't a signal it's a proc. if you want window_smoothing to react to movement then it should explicitly listen for movement and remove/readd itself in that condition.

Reply:
@LemonInTheDark done